gradle.properties

```diff
@@ -1,6 +1,6 @@
 org.gradle.parallel=true
 
 group=io.pivotal.cfenv
-version=3.1.6-SNAPSHOT
+version=3.3.0-SNAPSHOT
```

Comment:
but this version does not have to be aligned with spring boot version though?
I was by coincidence that they kinda matched.

Reply:
I will change back to 3.1.6-SNAPSHOT.